Rhinoceros_Engine/Convert/ToBHoM.cs

```diff
@@ -245,7 +245,7 @@ public static BHG.ICurve ToBHoM(this RHG.NurbsCurve rCurve)
 
         /***************************************************/
 
-        public static BHG.ICurve ToBHoM(this RHG.Curve rCurve)
```

Comment:
Uhm, is there a specific non-workaroundable reason why you need this?

Reply:
The reason to use the Boolean was that the 360deg arcs cannot be converted to circles when they are trims. I resolved it with a private method and a bit of boiler plate finally, as I also found this Boolean awful 😨.